### PR TITLE
DP notification at CN side submodule and Arion-DP benchmark update

### PR DESCRIPTION
**Pull request by w-yue**
This PR consists of two components for 930:
1. DP notification at CN side submodule;
2. Arion-DP benchmark update with newly added Redis benchmarking results.

It also includes some code clean up and optimizations.

### CODE REVIEW

File: cn/cn_dpnd/cn_dpnd_usr.c [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    /* Open RAW socket to send on */
+    if ((sockfd = socket(AF_INET, SOCK_DGRAM, 0)) < 0) {
+        printf("DPA failed to open raw socket! Err: %s\n", strerror(errno));
+        return 1;
+    }
+    printf("Successfully created a socket.\n");
+    for(;;){
+        //printf("One round in the forever for loop...\n");
+        dp_encap_opdata_t opdata;
+        unsigned char src_mac[6] = {108,221,238,0,0,44}; //"6c:dd:ee:00:00:2c"
```
Comment (zzxgzgz) [inline cn/cn_dpnd/cn_dpnd_usr.c:228]: Should we keep these hard-coded lines?

Comment (w-yue) [reply cn/cn_dpnd/cn_dpnd_usr.c:228]: Ideally, they could be in a separate unit test code. Maybe we can do that later in another PR.

Comment (zzxgzgz) [reply cn/cn_dpnd/cn_dpnd_usr.c:228]: I agree, let's do it then. 

lgtm.

File: .gitmodules [hunk 2]
```diff
@@ -4,3 +4,6 @@
 [submodule "src/extern/alcor-control-agent"]
```
Comment (zzxgzgz) [inline .gitmodules:4]: LGTM

**Review by zzxgzgz — APPROVED** [review]
LGTM

**Pull request merged**